src/Tracking/TrackingData/DonationFormsData.php

```diff
@@ -81,6 +83,7 @@ protected function getData()
                 'form_template' => $this->getFormTemplate($formId),
                 'donor_count' => $this->formDonorCounts[$formId],
                 'revenue' => $this->formRevenues[$formId],
+                'event_block' => FormBlocks::formId($formId)->hasBlock('givewp/event-tickets'),
```

Comment:
After some troubleshooting on a live site, it seems that this `bool` needs to be manually cast to an `int` (`1` or `0`) before being sent to the server (which the server will then re-cast as a `bool`).

Reply:
```suggestion
                'event_block' => (int) FormBlocks::formId($formId)->hasBlock('givewp/event-tickets'),
```